shipping/uc_usps/uc_usps.install

```diff
@@ -97,12 +97,157 @@ function uc_usps_update_1000() {
+    'UGAMS' => 'USPS Ground Advantage Machinable Single-piece',
+  ));
+  $config->set('uc_usps_newapi_dom_parcel_all_services', array(
+    'FCLM' => 'First Class Letter Metered',
+    'FCF' => 'First Class Flats',
+    'PMMSFRB' => 'Priority Mail Machinable Small Flat Rate Box',
+    'PMMMFRB' => 'Priority Mail Machinable Medium Flat Rate Box',
+    'PMMLFRB' => 'Priority Mail Machinable Large Flat Rate Box',
+    'PMMLFRBA' => 'Priority Mail Machinable Large Flat Rate Box APO/FPO/DPO',
+    'PMMS' => 'Priority Mail Machinable Single-piece',
+    'PMMCNPT1' => 'Priority Mail Machinable Cubic Non-Soft Pack Tier 1',
+    'PMMCSPT1' => 'Priority Mail Machinable Cubic Soft Pack Tier 1',
+    'PMEMS' => 'Priority Mail Express Machinable Single-piece',
+    'UGAMS' => 'USPS Ground Advantage Machinable Single-piece',
+    'UGAMCNPT1' => 'USPS Ground Advantage Machinable Cubic Non-Soft Pack Tier 1',
+    'UGAMCSPT1' => 'USPS Ground Advantage Machinable Cubic Soft Pack Tier 1',
+    'BPMMP' => 'Bound Printed Matter Machinable Presorted',
+    'BPMNP' => 'Bound Printed Matter Nonstandard Presorted',
+    'LMNB' => 'Library Mail Nonstandard Basic',
+    'LMM5' => 'Library Mail Machinable 5-digit',
+    'LMN5' => 'Library Mail Nonstandard 5-digit',
+    'MMNB' => 'Media Mail Nonstandard Basic',
+    'MMM5' => 'Media Mail Machinable 5-digit',
+    'MMN5' => 'Media Mail Nonstandard 5-digit',
+  ));
+  $config->set('uc_usps_newapi_intl_env_all_services', array(
+    'FCLM' => 'First Class Letter Metered',
+    'FCP' => 'First Class Postcards',
+    'FCF' => 'First Class Flats',
+    'FPISMIS' => 'First-Class Package International Service Machinable ISC Single-piece',
+    'PMIIFRE' => 'Priority Mail International ISC Flat Rate Envelope',
+    'PMIMIPFRE' => 'Priority Mail International Machinable ISC Padded Flat Rate Envelope',
+    'PMIIS' => 'Priority Mail International ISC Single-piece',
+    'PMIMIS' => 'Priority Mail International Machinable ISC Single-piece',
+    'PMIILFRE' => 'Priority Mail International ISC Legal Flat Rate Envelope',
+    'PMEIIFRE' => 'Priority Mail Express International ISC Flat Rate Envelope',
+    'PMEIILFRE' => 'Priority Mail Express International ISC Legal Flat Rate Envelope',
+    'PMEIIS' => 'Priority Mail Express International ISC Single-piece',
+    'PMEIIPFRE' => 'Priority Mail Express International ISC Padded Flat Rate Envelope',
+  ));
+  $config->set('uc_usps_newapi_intl_parcel_all_services', array(
+    'FPISMIS' => 'First-Class Package International Service Machinable ISC Single-piece',
+    'PMIMILFRB' => 'Priority Mail International Machinable ISC Large Flat Rate Box',
+    'PMIMIMFRB' => 'Priority Mail International Machinable ISC Medium Flat Rate Box',
+    'PMIIS' => 'Priority Mail International ISC Single-piece',
+    'PMIMIS' => 'Priority Mail International Machinable ISC Single-piece',
+    'PMEIIS' => 'Priority Mail Express International ISC Single-piece',
+  ));
+
+  $config->save();
+}
+
 /**
```

Comment:
It's an odd choice to put update hooks *above* the more "stable" functions. I suggest moving them to the bottom which is more typical since there's potentially unlimited number.